manim/mobject/mobject.py

```diff
@@ -66,6 +82,14 @@ class Mobject(Container):
 
     """
 
+    animation_overrides: Dict[
+        Type["Animation"], Callable[["Mobject"], "Animation"]
+    ] = {}
```

Comment:
Don't think this should be here now that classes store their own overrides.
```suggestion
```

Reply:
In my opinion the annotation helps to understand the structure how the overrides are saved, but I probably can move it somewhere else.

Reply:
A comment above `cls.animation_overrides = {}` (or even a type annotation there) seems appropriate to me

Reply:
Yes, that's what I wanted to do as well